common/scala/src/main/scala/org/apache/openwhisk/core/containerpool/logging/DockerToActivationFileLogStore.scala

```diff
@@ -123,7 +123,16 @@ class DockerToActivationFileLogStore(system: ActorSystem, destinationDirectory:
                            container: Container,
                            action: ExecutableWhiskAction): Future[ActivationLogs] = {
 
-    val logs = container.logs(action.limits.logs.asMegaBytes, action.exec.sentinelledLogs)(transid)
+    // wait for a sentinel only if no container (developer) error occurred to avoid
+    // that log collection continues if the action code still logs after timeout
+    val isContainerTimeoutError = activation.response.isContainerTimeoutError(action.limits.timeout.duration)
```

Comment:
Can any of these vals reference the same vals from `DockerToActivationLogStore.scala`?

Reply:
done, we introduced common code (`logStream`) in `DockerToActivationLogStore` and we added `isTimedoutActivation` function to `WhiskActication`